styles/app.scss

```diff
@@ -1,6 +1,41 @@
 // @import 'yoga';
 // @import 'normalize';
-
 // $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
 // @import 'font-awesome';
 @import 'box-sizing';
+@import '_shop-items';
+@import 'variables';
+body {
+
+}
+
+.outer-container {
+    padding: $gutter;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+}
+.container {
+    width: 100%;
+    max-width: $max-content-width;
+    margin: auto;;
+}
+.row {
+    display: flex;
+    flex-flow: wrap;
+
+    @media screen and (max-width: 48rem){
```

Comment:
Having a `max-width` and `min-width` will make this exercise a lot easier. Also put your media query outside of this `.row` and it may make things easier to organize.